ui/options_plugins.ui

```diff
@@ -98,7 +98,7 @@
             </sizepolicy>
            </property>
            <property name="text">
-            <string>Download plugins</string>
+            <string>Reload available plugins</string>
```

Comment:
How should i name it ?
It is "reload the information about available and installed plugins", so something like "Reload plugin infos" could do ?